samples/RoutingSample.Web/RoutingSample.Web.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
```

Comment:
We shouldn't depend on common in samples. Import dependency.props instead and add the explicity reference to NETStandard.Library.NETFramework